src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  TouchableOpacity,
+  View,
+  StyleSheet,
+  TouchableOpacityProps,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+  Interaction,
+} from '@kitten/theme';
+import { CheckMark } from '@kitten/ui/drawable/checkmark/checkmark.component';
+
+interface CheckBoxProps {
+  checked?: boolean;
+  onChange?: (checked: boolean) => void;
+  appearance?: string | 'default';
+  status?: string | 'error';
+  size?: string | 'big' | 'small';
```

Comment:
Please remove exact values

Reply:
currently the same syntax is used in [radio component](https://github.com/akveo/react-native-ui-kitten/blob/next/src/framework/ui/radio/radio.component.tsx#L14)
we can refactor this in each implemented component in future